extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -47,6 +47,9 @@ class AmpStickyAd extends AMP.BaseElement {
     /** @const @private {!Element} */
     this.ad_ = children[0];
 
+    /** @private @const {!Element|null} */
+    this.closeButton_ = null;
```

Comment:
I don't believe we need this reference anymore since we're not explicitly unbinding the event listener.